src/components/views/ViewDataTable/index.tsx

```diff
@@ -77,6 +96,40 @@ const ViewDataTable: FunctionComponent<ViewDataTableProps> = ({ columns, rows, v
         },
     });
 
+    // TODO: Create mutation using new factory pattern
+    const addRowMutation = useMutation(async (person: ZetkinPerson) => {
+        const res = await defaultFetch(`/orgs/${orgId}/people/views/${view.id}/rows/${person.id}`, {
+            method: 'PUT',
+        });
+        const data = await res.json();
+        return data.data;
+    }, {
```

Comment:
Why not just make a `viewRowsResource` with just this handler? It doesn't take very long at all! You can add the onsetter handler to the mutation call.

Reply:
Good point! At the time that I wrote this a week ago it hadn't been merged yet. That's actually why I left it "inline" like this rather than creating a new module, because I knew that we would want to refactor it. And now that the resource factory code has been merged, I agree I should be using it here. Will fix!